testsuite/tests/kuadrant/authorino/caching/metadata/test_caching.py

```diff
@@ -28,21 +28,23 @@ def test_cached(client, auth, module_label, mockserver):
         - only single external value evaluation occurs. The second response contains cached (in-memory) value
     """
     response = client.get("/get", auth=auth)
-    data = extract_from_response(response, module_label, "uuid")
+    data = extract_response(response)[module_label]["uuid"] % None
     response = client.get("/get", auth=auth)
-    cached_data = extract_from_response(response, module_label, "uuid")
+    cached_data = extract_response(response)[module_label]["uuid"] % None
 
+    assert data is not None
+    assert cached_data is not None
     assert data == cached_data
     assert len(mockserver.retrieve_requests(module_label)) == 1
 
 
 def test_cached_ttl(client, auth, module_label, cache_ttl, mockserver):
     """Tests that cached value expires after ttl"""
     response = client.get("/get", auth=auth)
-    data = extract_from_response(response, module_label, "uuid")
+    data = extract_response(response)[module_label]["uuid"] % None
     sleep(cache_ttl)
     response = client.get("/get", auth=auth)
-    cached_data = extract_from_response(response, module_label, "uuid")
+    cached_data = extract_response(response)[module_label]["uuid"] % None
 
     assert data != cached_data
```

Comment:
Assert if `data` and `cached_data` are `not None`, same as in the previous test-case